sympy/physics/vector/point.py

```diff
@@ -215,14 +219,24 @@ def locatenew(self, name, value):
         self.set_pos(p, -value)
         return p
 
-    def pos_from(self, otherpoint):
+    def pos_from(self, otherpoint, frame=None, variables=False, dcm_connected=False):
         """Returns a Vector distance between this Point and the other Point.
 
         Parameters
         ==========
 
         otherpoint : Point
             The otherpoint we are locating this one relative to
+        frame : ReferenceFrame
+            Expresses the resulting vector in this frame
```

Comment:
Initial reaction is that we should leave this to the `express()` method on `Vector`. Why is it necessary to have this here?